api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -143,7 +143,7 @@ public Workspace apply(org.pmiops.workbench.db.model.Workspace workspace) {
 
   private static final ResearchPurpose createResearchPurpose(org.pmiops.workbench.db.model.Workspace workspace) {
     ResearchPurpose researchPurpose = new ResearchPurpose()
-        .diseaseFocusedResearch(workspace.getDiseaseFocusedResearch())
+        .diseaseFocusedResearch(workspace.getDiseaseOfFocus() != null && !workspace.getDiseaseOfFocus().isEmpty())
```

Comment:
Since we're ignoring this field from the client now, would it make sense to remove it from workbench.yaml?

Reply:
Its actually still getting used on the about page for the workspace.

Reply:
You might want to add a `trim()` to this in case `" "` makes it through.

Reply:
Done!